docs/searching-algorithms/interpolation-search.md

```diff
@@ -0,0 +1,260 @@
+---
+title: Interpolation Search
+slug: /searching-algorithms/interpolation-search
+---
+
+## Introduction
+
+Interpolation Search is a technique to search a key in an array consisting of sorted and uniformly distributed values. By Uniformly distributed we mean, the difference between two adjacent elements must be consistent. For example:
+
+**Sorted but non-uniformly distributed array:**
+
+The following array is sorted but not uniform distributed as the difference between adjacent elements is not the same.
```

Comment:
Keep it `The following array is sorted but not uniformly distributed as the difference between adjacent elements is not the same.`

Reply:
Also use a colon `:` after `not the same`